pyproject.toml

```diff
@@ -67,9 +68,42 @@ features = [
   "ci",
 ]
 
+[tool.ruff]
+line-length = 99
+extend-include = ["pyproject.toml", "lib/spack/**/*.py", "var/spack/repos/**/*.py", "bin/spack"]
```

Comment:
pyproject.toml can't be formatted. All .py files are included by default. I think you only need to add bin/spack.

Reply:
done